kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jSinkConnectorConfig.kt

```diff
@@ -71,6 +74,8 @@ class Neo4jSinkConnectorConfig(originals: Map<*, *>): Neo4jConnectorConfig(confi
         const val TOPIC_CDC_SCHEMA = "neo4j.topic.cdc.schema"
         const val TOPIC_CUD = "neo4j.topic.cud"
 
+        const val MERGE_PROPERTIES_ENABLED = "neo4j.merge.properties.enabled"
```

Comment:
```suggestion
        const val TOPIC_PATTERN_MERGE_NODE_PROPERTIES_ENABLED = "neo4j.topic.pattern.merge.node.properties.enabled"
        const val TOPIC_PATTERN_MERGE_RELATIONSHIP_PROPERTIES_ENABLED = "neo4j.topic.pattern.merge.relationship.properties.enabled"
```

And please adapt all accordingly.